editors/code/package.json

```diff
@@ -268,6 +268,125 @@
                 },
                 "pattern": "$rustc"
             }
+        ],
+        "colors": [
+            {
+                "id": "ralsp.background",
+                "description": "Background color",
+                "defaults": {
+                    "dark": "#3F3F3F",
+                    "light": "#001080",
+                    "highContrast": "#000000"
+                }
+            },
+            {
+                "id": "ralsp.comment",
+                "description": "Color for comments",
+                "defaults": {
+                    "dark": "#7F9F7F",
+                    "light": "#008000",
+                    "highContrast": "#7CA668"
+                }
+            },
+            {
+                "id": "ralsp.string",
+                "description": "Color for strings",
+                "defaults": {
+                    "dark": "#CC9393",
+                    "light": "#A31515",
+                    "highContrast": "#CE9178"
+                }
+            },
+            {
+                "id": "ralsp.unsafe",
+                "description": "Color for unsafe",
+                "defaults": {
+                    "dark": "#FF3030",
+                    "light": "#FF1010",
+                    "highContrast": "#FF1010"
+                }
+            },
+            {
+                "id": "ralsp.keyword",
+                "description": "Color for keywords",
+                "defaults": {
+                    "dark": "#F0DFAF",
+                    "light": "#0000FF",
+                    "highContrast": "#569CD6"
+                }
+            },
+            {
+                "id": "ralsp.control",
```

Comment:
Is there an easy way to allow both

* highligting all keywords the same (this is what I personally prefer)
* highlight various different classes of keywords differently

Perhaps we should just name classes `ralsp.keyword`, `ralsp.keyword.controlflow`, `ralsp.keyword.unsafe`?

Reply:
Updated. I used `ralsp.keyword.control` to match the default theme (`keyword.control`).